types/ws/index.d.ts

```diff
@@ -282,7 +281,7 @@ declare namespace WebSocket {
         backlog?: number | undefined;
         server?: HTTPServer | HTTPSServer | undefined;
         verifyClient?: VerifyClientCallbackAsync | VerifyClientCallbackSync | undefined;
-        handleProtocols?: any;
+        handleProtocols?: (protocols: Set<string>, request: IncomingMessage) => string | false;
```

Comment:
This is technically a breaking change as `protocols` is an array in `ws@<8`.

Reply:
A `@types` definition file should be used with the same `major.minor` version. This PR will be used only with `ws@8`. The change would not be an issue.